frontend/src/routes/candidate/(protected)/questions/all/+page.svelte

```diff
@@ -0,0 +1,121 @@
+<script lang="ts">
+  import {page} from '$app/stores';
+  import {Button} from '$lib/components/button';
+  import {BasicPage} from '$lib/templates/basicPage';
+  import {Expander} from '$lib/components/expander';
+  import LikertResponseButtons from '$lib/components/questions/LikertResponseButtons.svelte';
+  import {answerContext} from '$lib/utils/answerStore';
+  import {goto} from '$app/navigation';
+  import Icon from '$lib/components/icon/Icon.svelte';
+  import {_} from 'svelte-i18n';
+
+  /**
+   * A small delay before moving to the unaswered question.
+   * TODO: Make this a global variable used throughout the app.
+   */
+  const DELAY_M_MS = 350;
+
+  const questions = $page.data.questions;
+  const categories = [...new Set(questions.map((question) => question.category))];
+
+  const store = answerContext.answers;
+  $: answerStore = $store;
+
+  let nofUnasweredQuestions = 0;
+  $: nofUnasweredQuestions = questions.length - Object.entries(answerStore).length;
+
+  function gotoToQuestion(id: string) {
+    const questionUrl = $page.url.pathname.replace(/\/all$/, '/' + id);
+    setTimeout(() => goto(questionUrl), DELAY_M_MS);
+  }
+</script>
+
+<BasicPage title={$_('candidateApp.allQuestions.title')}>
+  <svelte:fragment slot="note">
+    <div class="text-warning">
+      <Icon name="important" />
+      {$_('candidateApp.allQuestions.warning', {values: {nofUnasweredQuestions}})}
+    </div>
+  </svelte:fragment>
+
+  <p class="text-center">
+    {$_('candidateApp.allQuestions.info')}
+  </p>
+
+  <p>
+    {#each categories as category}
+      <div class="edgetoedge-x">
+        <Expander title={category || ''} variant="category">
+          {#each questions as question, i}
+            <!-- Show questions based on categories -->
+            {#if category === question.category}
+              <!-- Question has been answered -->
+              {#if answerStore[question.id]}
+                <div class="pb-20 pt-20">
+                  <div class="text-accent">
+                    {question.category}
+                  </div>
+
+                  <Expander title={question.text || ''} variant="question">
+                    {question.info}
+                  </Expander>
+
+                  <div class="pt-10">
+                    <!-- This gives empty form label error for every empty dot, but fix should come from LikertResponseButton -->
```

Comment:
Could you specify that it is an error in the wave extension? It is not apparent from the context :D